content/en/docs/Tasks/creating-operator-manifests.md

```diff
@@ -268,15 +268,14 @@ However, the following is a suggested guideline to follow when including Priorit
 #### Extension apiservers and APIServices
 
 Please see the document on [extension apiservers]() if your operator does not rely on CRDs to provide its API.
```

Comment:
I know this is not from your change-set but do we have a link for extension apiservers? Otherwise this goes nowhere.

Reply:
Unfortunately we don't have any document on extension apiservers atm. This has to be put on our to-do list.